docs/source/write_first_app.rst

```diff
@@ -160,6 +160,12 @@ Next, let's prepare the sample Asset Transfer Javascript application that will b
 
 - `JavaScript application <https://github.com/hyperledger/fabric-samples/blob/{BRANCH}/asset-transfer-basic/application-javascript>`__
 
+Note that the sample application is also available in Go and Java at the links below:
+
+- `Go application <https://github.com/hyperledger/fabric-samples/blob/{BRANCH}/asset-transfer-basic/application-go>`__
```

Comment:
We don't create release branches for fabric-samples anymore, so I guess all of these should simply use 'master' rather than {BRANCH}.
Same for the other fabric-samples reference above.

Reply:
Done.